homeassistant/components/openai_conversation/conversation.py

```diff
@@ -142,11 +127,23 @@ async def async_process(
                         device_id=user_input.device_id,
                     )
 
-                    prompt = (
-                        await llm_api.async_get_api_prompt(empty_tool_input)
-                        + "\n"
-                        + prompt
+                    api_prompt = await llm_api.async_get_api_prompt(empty_tool_input)
+
+                else:
+                    api_prompt = llm.PROMPT_NO_API_CONFIGURED
+
+                prompt = (
```

Comment:
Maybe this could be a helper since its the same as https://github.com/home-assistant/core/pull/118082/files#diff-3422a8bc6ee6bb5c00835c1874d304e0c213db6c8f67b67aa2308b3528da5055R225

Reply:
Both integrations are very similar. I want to see how they both evolve before extracting helpers.